src/components/WalletFace/Actions/WalletFaceActions.js

```diff
@@ -71,16 +72,28 @@ class WalletFaceActions extends PureComponent<Props> {
   render() {
     const {
       toggle,
-      balance,
+      // balance,
       isToggled,
     }: Props = this.props
 
     return (
-      <div className={classNames('wallet-face-actions', isToggled && '-toggled')}>
-        <div onClick={toggle} className='overlay' />
-        {isToggled ? (
-          <div className='actions'>
-            {balance && (
+      <div className={classNames(
+        walletFaceActions.core,
+        isToggled && walletFaceActions['-toggled'],
```

Comment:
we stopped using of rscss namings, because we don't need modifiers with css modules (styles have plain structure).
try to edit old class like `variants` and `elements` from rscss.